src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicSingleBlackHole.hpp

```diff
@@ -110,11 +151,21 @@ struct EvolutionMetavars
 
   using typename gh_base::step_actions;
 
+  // Add ElementInitInterpPoints to initialization_actions.
+  using helper_init =
```

Comment:
call this `initialization_actions` and remove line 150. Instead, use `gh_base::initialization_actions` in this implementation.